src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -617,6 +646,70 @@ async Task<bool> OnPause(SessionId sessionId, JObject args, CancellationToken to
             return true;
         }
 
+        async Task<MethodInfo> LoadSymbolsOnDemand(AssemblyInfo asm, uint method_token, SessionId sessionId, CancellationToken token)
+        {
+            foreach (var urlSymbolServer in urlSymbolServerList) 
+            {
+                MethodInfo method = null;
+                var context = GetContext(sessionId);
+                ImageDebugHeader header = asm.Image.GetDebugHeader();
+                for (var i = 0; i < header.Entries.Length; i++)
+                {
+                    var entry = header.Entries[i];
+                    if (entry.Directory.Type != ImageDebugType.CodeView)
+                    {
+                        continue;
+                    }
+
+                    var data = entry.Data;
+
+                    if (data.Length < 24)
+                        return null;
+
+                    var pdbSignature = (data[0]
+                        | (data[1] << 8)
+                        | (data[2] << 16)
+                        | (data[3] << 24));
+
+                    if (pdbSignature != 0x53445352) // "SDSR"
```

Comment:
nit: Is there any documentation/spec/code link that we can provide for the signature/guid extraction stuff here?

Reply:
I copied this from https://github.com/mono/mono/blob/45d996b7ba059ad88d9b3f74182e3a9da16da34d/mono/metadata/debug-mono-ppdb.c#L101

Reply:
Great. Just a reference to the relative file path, in a comment would be useful.

Reply:
Done! 👍